src/components/Notifications/Form/__tests__/IntegrationRecipientTypeahead.test.tsx

```diff
@@ -150,4 +150,23 @@ describe('src/components/Notifications/Form/IntegrationRecipientTypeAhead', () =
 
         expect(onSelected).toHaveBeenCalled();
     });
+
+    it('Integration recipients that have been previously used in the form are disabled', async () => {
+        // render(<IntegrationRecipientTypeahead
```

Comment:
You could probably wrap this with `<Formik>`, and use an initial state (a behavior group that has the actions that we want to test).

See the [behavior group save modal](https://github.com/RedHatInsights/notifications-frontend/blob/9d3010e92951234db43e98892a935cc51f47a980/src/components/Notifications/BehaviorGroup/BehaviorGroupSaveModal.tsx#L60).